graphics/osxphotos/Portfile

```diff
@@ -26,9 +26,9 @@ long_description        {*}${description}
 
 homepage                https://github.com/RhetTbull/osxphotos
 
-checksums               rmd160  99472b43ebfd96cd5a431b480f067ee525a92da7 \
-                        sha256  f8a650a41bff5ec3feac86fc765ca610a728c31d70dfba3880cca9412a696d87 \
-                        size    2085752
+checksums               rmd160  dddaa5ed50fc2d4ae8f52cf9b213a50f347a774a \
+                        sha256  9784a6da9232bc26c6780845625b7685e4732370efa1e7952fc6233a1688838a \
+                        size    2085894
 
 python.default_version  311
```

Comment:
this ideally should be using `312` now as that is MacPorts' default since January 1st.

Reply:
@breun Indeed. If you can update that, we can merge this quickly. Thank you as always for maintaining the ports you handle.

Reply:
I see that `py312-bitmath` is not available yet, so I maybe we can merge this update as-is for now? I'll bump the Python version later when all dependencies are available for Python 3.12.

Reply:
I will merge it. If you could work on getting the dependencies updated for 3.12 and then this, it would be appreciated.

Reply:
I don’t maintain the dependency ports, but I tried adding a Python 3.12 version for the bitmath dependency locally, but that in turn required another Python dependency that doesn’t have a Python 3.12 version yet. I’ll try upgrading osxphotos to Python 3.12 again with future updates.

Reply:
You don't have to maintain something to do an independent pull request for it. Most of the updates to the tree are not from maintainers these days.

Reply:
I know, but I don't feel comfortable enough with these dependency ports to add versions for new Python versions. I've been burned too often due to Python's questionable backwards compatibility across minor versions.

Reply:
For the ones that have tests, it should be straightforward to try them out.